source/blog/2018-02-06-fedora-atomic-workstation.html.md

```diff
@@ -0,0 +1,172 @@
+---
+title: An Introduction to Fedora Atomic Workstation
+author: jlebon
+date: 2018-02-06 00:00:00 UTC
+published: true
+comments: false
+tags: atomic, fedora, workstation, desktop
+---
+
+For the last few Fedora releases, the Workstation WG has been working on
+combining the best of the Project Atomic pattern with the Fedora Workstation
+Edition into a deliverable dubbed &quot;Fedora Atomic Workstation&quot; (often
+shortened as &quot;Atomic WS&quot;). In Fedora 27, we have reached a point where
```

Comment:
I think for this blog it would be great if we didn't abbreviate it to WS.

Reply:
Hmm, I think it flows nicer with the abbreviation, no? It gets really tedious to read "Fedora Atomic Workstation" over and over otherwise. :)

Reply:
ehh. I think for this post it would be better if we are repetitive from a branding perspective. Once we have beat it into our heads and their heads quite a bit I think we can start to abbreviate. Even so I still don't really abbreviate Atomic Host as AH when I do blog posts, but mostly for informal mails to mailing lists.